xinference/core/supervisor.py

```diff
@@ -870,6 +870,9 @@ async def _check_dead_nodes(self):
                             address,
                             dead_models,
                         )
+                        for model_uid in dead_models:
+                            self._model_uid_to_replica_info.pop(model_uid[:-4], None)
```

Comment:
Do not use `model_uid[:-4]` to get `model_uid` from `rep_model_uid`.
Use `parse_replica_model_uid` util to get real `model_uid`.